kernel/reduction.ml

```diff
@@ -974,16 +974,11 @@ let gen_conv cv_pb l2r reds env evars univs t1 t2 =
       let _ = clos_gen_conv reds cv_pb l2r evars env univs (univs, checked_universes) t1 t2 in
         ()
 
-(* Profiling *)
-let gen_conv cv_pb ?(l2r=false) ?(reds=TransparentState.full) env ?(evars=(fun _->None)) =
+let gen_conv cv_pb ?l2r ?reds env ?evars =
```

Comment:
You should probably even merge this trivial wrapper with the above function.

Reply:
Done, I actually wondered about this one but didn't want to be too invasive on API's I don't use.